### DISCUSSION TITLE
Sync not stopping?

### DISCUSSION THREAD

**Discussion by aagha** [thesamim/TickTickSync#163] [Q&A]
After my [previous post](https://github.com/thesamim/TickTickSync/discussions/162), I decided to clean stuff up, so I uninstalled the plugin from Obsidian, and have been tweaking my tasks in TickTick.

However, it seems that no matter what I do to remove the link to the task locaiton (e.g., Watch Jaws), the task keeps re-syncing an coming back with the link!

![image](https://github.com/user-attachments/assets/17889ac2-9ecb-4fcd-b8fd-9c614609c720)

I've stopped the (Windows) desktop app and am working in the web client, but every few minutes, the task comes back with the link.

Is there a way to see if the app I autorized in Obsidian is _unauthorized_ or somehow stop the syncing that seems to be happening every few minutes?

Note, this is for just some  tasks, not al

**Comment by thesamim** [comment]
@aagha : I'm not sure I understand the issue.

Removing the link will not affect processing one or the other. The plugin looks for the #ticktick tag and syncs the task bi-directionally. The only way to stop the syncing is to remove the #ticktick tag from the Obsidian side. Unfortunate side effect: The plugin will attempt to delete it from TickTick. The plugin does not cover the use case of wanting to stop the synchronization of a particular task.

> Is there a way to see if the app I autorized in Obsidian is unauthorized or somehow stop the syncing that seems to be happening every few minutes?

Not sure what you mean by authorized. The plugin is either logged in or it's not. The plugin only runs when it is enabled and Obsidian is running. 

What am I missing?
